source/solvers/navier_stokes_base.cc

```diff
@@ -574,36 +616,33 @@ NavierStokesBase<dim, VectorType, DofsType>::refine_mesh_kelly()
   // Time monitoring
   TimerOutput::Scope t(this->computing_timer, "refine");
 
-  Vector<float>       estimated_error_per_cell(tria.n_active_cells());
-  const MappingQ<dim> mapping(
-    this->velocity_fem_degree,
-    this->simulation_parameters.fem_parameters.qmapping_all);
+  Vector<float> estimated_error_per_cell(tria.n_active_cells());
   const FEValuesExtractors::Vector velocity(0);
   const FEValuesExtractors::Scalar pressure(dim);
   auto &                           present_solution = this->present_solution;
   if (this->simulation_parameters.mesh_adaptation.variable ==
       Parameters::MeshAdaptation::Variable::pressure)
     {
       KellyErrorEstimator<dim>::estimate(
-        mapping,
+        *this->velocity_mapping,
```

Comment:
I guess that to be consistent this should be: 
```suggestion
        *this->pressure_mapping,
```
right?